app/src/organisms/SendProtocolToOT3Slideout/index.tsx

```diff
@@ -38,7 +38,7 @@ export function SendProtocolToOT3Slideout(
     srcFiles,
     mostRecentAnalysis,
   } = storedProtocolData
-  const { t } = useTranslation(['protocol_details', 'shared'])
+  const { t } = useTranslation(['protocol_details', 'protocol_list', 'shared'])
```

Comment:
`shared` can be removed from the array since it isnt in use

```suggestion
  const { t } = useTranslation(['protocol_details', 'protocol_list'])
```